### ISSUE TITLE
[Bug] Error when trying to import Launcher XML

### ISSUE THREAD

**Issue by amura11** [Wintermute0110/plugin.program.AEL#149]
When trying to import a launcher XML from the [AEL Assets Library](https://github.com/Wintermute0110/AEL-asset-library) Kodi throws an error. I've attached the logs but the error is:
```
Error Type: <class 'NameError'>
Error Contents: name 'nfo_dic' is not defined
```

**Steps to Reproduce**
1. Download the AEL Asset Library
2. Open Kodi
2. Navigate to *AEL* > *Utilites*
3. Run the import utility
4. Select a Launcher XML and click Ok

**System**
- Kodi Matrix
- Ubuntu 20.04

**Logs:** [ael-error.log](https://github.com/Wintermute0110/plugin.program.AEL/files/6928711/ael-error.log)

**Comment by Wintermute0110** [comment]
I think that has been fixed in the current version in Github. Please try and remember to select the appropriate branch.

**Comment by amura11** [comment]
Sorry I should have put the plugin version in there. I have the 0.10.0
I'll give the development version a try when I have some time

**Comment by Wintermute0110** [comment]
I just released `0.10.1`, try that version.

**Comment by amura11** [comment]
That fixed it, thank you!
